app/parse.py

```diff
@@ -15,9 +39,61 @@ class Product:
     num_of_reviews: int
 
 
+def parse_single_product(product: WebElement):
+    return Product(
+        title=product.find_element(By.CLASS_NAME, "title").text,
+        description=product.find_element(By.CLASS_NAME, "description").text,
+        price=float(product.find_element(By.CLASS_NAME, "price").text.replace("$", "")),
```

Comment:
Consider adding error handling for the conversion of price to float. If the price text is not in the expected format, this could raise a ValueError.